ensime-expand-region.el

```diff
@@ -0,0 +1,61 @@
+;;; ensime-expand-region.el --- Ensime expansions for Expand Region  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2015  Sebastian Wiesner
+
+;; Author: Sebastian Wiesner <swiesner@lunaryorn.com>
+
+;; This file is not part of GNU Emacs
```

Comment:
I'm ok with the shorter header form of `ensime-http`. This classic form is so verbose...